pkg/asset/installconfig/azure/session.go

```diff
@@ -77,39 +115,22 @@ func newSessionFromFile(authFilePath string, cloudName azure.CloudEnvironment) (
 		return nil, errors.Wrap(err, "failed to get settings from file")
 	}
 
-	authSettings.Values[auth.ActiveDirectoryEndpoint] = env.ActiveDirectoryEndpoint
```

Comment:
I believe we no longer need this line as we explicitly set endpoint in `auth.ClientCredentialsConfig` now.